search-commons/src/main/java/no/unit/nva/search2/model/PagedSearchResourceDto.java

```diff
@@ -36,6 +38,11 @@ private PagedSearchResourceDto(Builder builder) {
         );
     }
 
+    public String toJson() {
```

Comment:
Use toJsonString() from implementing JsonSerializable?

Reply:
trouble with intellij  records and implements.... cannot do :-(

Reply:
What if you just do the `implements JsonSerializable` without the `toJsonString` override method, let the interface handle it itself?